src/components/Paragraph.tsx

```diff
@@ -0,0 +1,16 @@
+import { BodyLong, Label } from "@navikt/ds-react";
+import React from "react";
+
+interface ParagraphProps {
+  label: string;
+  body: string;
+}
+
+export const Paragraph = ({ label, body }: ParagraphProps) => {
```

Comment:
Flyttet ut som en felles komponent og bruker den fra arbeidsuførhet- og aktivitetskrav-historikk.

Reply:
Noe for dialogmøte-historikk og, eller ikke?

Reply:
Den er bygget opp litt annerledes basert på `DocumentComponent` så krever i så fall en større omskriving, men absolutt rom for forbedring der hvis vi skal jobbe med dialogmøte-sidene :+1: